src/boundaries/stagger_and_step.jl

```diff
@@ -0,0 +1,217 @@
+export stagger_and_step!, stagger_trajectory!
+using LinearAlgebra:norm
+using Random
+using ProgressMeter
+
+
+"""
+    stagger_trajectory!(ds, x0, Tm, isinside; kwargs...) -> xi
+
+This function returns a point `xi` which _guarantees_ `T(xi) > 
+Tm` with a random walk search around the initial coordinates 
+`x0`. `T(xi)` is the escape time of the initial condition `xi`.  
+There is no guarantee to find such point, the parameters 
+must be adjusted to find the suitable point. This is an .
+auxiliary function for [`stagger_and_step`](@ref). Keyword arguments and 
+definitions are identical for both functions. 
+
+The initial search radius `δ₀` is big, `δ₀ = 1.` by default.
+
+"""
+function stagger_trajectory!(ds, x0, Tm, isinside; δ₀ = 1., stagger_mode = :exp, max_steps = Int(1e5), f = 1.1)
+    T = escape_time!(ds, x0, isinside)
+    xi = deepcopy(x0) 
```

Comment:
```suggestion
    xi = copy(x0) 
```
why do you needed to deepcopy here? `x0` is at most an array.